cmd/glasskube/cmd/describe.go

```diff
@@ -148,10 +159,25 @@ func printRepositories(pkg *v1alpha1.Package, repos []v1alpha1.PackageRepository
 		fmt.Fprintf(os.Stderr, " * %v", repo.Name)
 		if isInstalledFrom(pkg, repo) {
 			fmt.Fprintln(os.Stderr, " (installed)")
+			return
+		}
+		fmt.Fprintln(os.Stderr)
+	}
+}
+
+func repositoriesAsMap(pkg *v1alpha1.Package, repos []v1alpha1.PackageRepository) []map[string]string {
+	repositories := []map[string]string{}
+	for _, repo := range repos {
+		repository := make(map[string]string)
+		repository["name"] = repo.Name
+		if isInstalledFrom(pkg, repo) {
+			repository["installed"] = "true"
 		} else {
-			fmt.Fprintln(os.Stderr)
+			repository["installed"] = "false"
 		}
+		repositories = append(repositories, repository)
 	}
+	return repositories
 }
```

Comment:
"installed" shoud be a bool. Please change this function, so it uses
```go
repository["installed"] = isInstalledFrom(pkg, repo)
```

You will have to change the signature of the function to `[]map[string]any` for this to work.